src/cdk/a11y/focus-monitor/focus-monitor.ts

```diff
@@ -388,51 +334,28 @@ export class FocusMonitor implements OnDestroy {
   }
 
   /**
-   * Sets the origin and schedules an async function to clear it at the end of the event queue.
-   * If the detection mode is 'eventual', the origin is never cleared.
+   * Updates the focus origin. If we're using immediate detection mode, we schedule an async
+   * function to clear the origin at the end of a timeout. The duration of the timeout depends on
+   * the origin being set.
    * @param origin The origin to set.
    */
-  private _setOriginForCurrentEventQueue(origin: FocusOrigin): void {
+  private _setOrigin(origin: FocusOrigin): void {
```

Comment:
I think `setOrigin` is a more accurate name. The name `_setOriginForCurrentEventQueue` implies that it _always_ sets the origin _for the current event queue_, which wasn't previously true and still isn't true.